cumulusci/core/utils.py

```diff
@@ -33,15 +34,27 @@ def parse_datetime(dt_str, format):
     return datetime(t[0], t[1], t[2], t[3], t[4], t[5], t[6], pytz.UTC)
 
 
-def process_bool_arg(arg):
-    """ Determine True/False from argument """
-    if isinstance(arg, bool):
-        return arg
+def process_bool_arg(arg: Union[int, str, None]):
+    """Determine True/False from argument.
+
+    None is considered false.
+    Similar to parts of the Salesforce API, there are a few true-ish and false-ish strings,
+        but "True" and "False" are the canonical ones."""
+    if isinstance(arg, (int, bool)):
+        return bool(arg)
+    elif arg is None:
+        # backwards compatible behaviour that some tasks
+        # rely upon.
+        return False
```

Comment:
I'm not quite following how this is the backwards-compatible behavior. Previously `None` would have failed the isinstance checks for bool and str, and returned None, like the test above showed.

Reply:
It's not a backwards-identical behaviour but I think it is close enough. I prefer that now this function will definitively return True or False, but since the return value for f(None) is still false-y, I doubt it will break any code.

Reply:
I like the change and agree it's pretty low risk. Just remove the comment here, which confused me because it seems to claim that this is the same return value as before.

Reply:
Oh, your other comment helped me figure out what you mean. You want to reject None as input, but are keeping it backwards compatible by interpreting it as falsy. Okay.